sourcegraph/cody
# Commands: Fix telemetry source usage

## Description

This PR:
- Removes the usage of `DefaultCodyCommands` for `source`
- Updates the source to be `args.source`. We default to `editor` because there are cases where these commands are called implicitly and we do not have control over passing custom arguments. (E.g. command palette, right click menu)

## Test plan

1. Run modified commands
2. Check source is correct
